client/src/redux/reducers/projects.reducers.js

```diff
@@ -186,6 +186,16 @@ export default function projects(state = initialState, action) {
           result: null,
         },
       };
+    case 'PROJECTS_SET_MEMBER_RESET':
```

Comment:
Is this case ever used?

Reply:
It is now! This was meant to run every time the modal is opened to clear out any cached errors.